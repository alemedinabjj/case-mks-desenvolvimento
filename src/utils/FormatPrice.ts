export const FormatPrice = (price: number) => {
  return new Intl.NumberFormat("pt-BR", {
    style: "currency",
    currency: "BRL",
  }).format(price);
};

//testing jest and react testing library

// Path: src/__tests__/FormatPrice.test.ts
