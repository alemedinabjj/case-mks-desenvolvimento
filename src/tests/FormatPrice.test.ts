import { FormatPrice } from "../utils/FormatPrice";

describe("FormatPrice", () => {
  it("should be able to format price", () => {
    const price = FormatPrice(1000);

    expect(price).toEqual("R$Â 1.000,00");

    expect(price).toMatchSnapshot();
  });
});
