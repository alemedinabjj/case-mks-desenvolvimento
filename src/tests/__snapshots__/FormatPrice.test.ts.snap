// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormatPrice should be able to format price 1`] = `"R$Â 1.000,00"`;
